### DISCUSSION TITLE
How to Modify the default grpc port 31337？

### DISCUSSION THREAD

**Discussion by vgi777** [BishopFox/sliver#1224] [Q&A]
“~/.sliver/configs/server.json” seems doesn't work

**Comment by moloch--** [comment]
The configuration file is only used if `daemon` mode is enabled via the configuration file.

If you used the Linux install script you'll want to modify the systemd service file, which by default will be installed to: `/etc/systemd/system/sliver.service`

Change this line:
```ExecStart=/root/sliver-server daemon```

To something like:
```ExecStart=/root/sliver-server daemon --lport 1234```

(and then restart the service / server process)
